src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllCodeRefactoringSuggestedAction.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Internal.Log;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Microsoft.CodeAnalysis.UnifiedSuggestions.UnifiedSuggestedActions;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    /// <summary>
+    /// Suggested action for fix all occurrences for a code refactoring.  Note: this is only used
+    /// as a 'flavor' inside CodeRefactoringSuggestionAction.
+    /// </summary>
+    internal sealed class FixAllCodeRefactoringSuggestedAction : SuggestedAction, IFixAllCodeRefactoringSuggestedAction
+    {
+        /// <summary>
+        /// The original code-action that we are a fix-all for.  This suggestion action
+        /// and our <see cref="SuggestedAction.CodeAction"/> is the actual action that 
+        /// will perform the fix in the appropriate document/project/solution scope.
+        /// </summary>
```

Comment:
i'm confused by this comment :)   You refer to two actions, but then say that both are the *actual action*.  Not sure what that means.